document/js-api/index.bs

```diff
@@ -387,7 +398,42 @@ A {{Module}} object represents a single WebAssembly module. Each {{Module}} obje
 
         Note: the validity and uniqueness checks performed during [=WebAssembly module validation=] ensure that each property name is valid and no properties are defined twice.
     1. Perform ! [=SetIntegrityLevel=](|exportsObject|, `"frozen"`).
-    1. Let |instanceObject| be a new {{Instance}} object whose internal \[[Instance]] slot is set to |instance| and the \[[Exports]] slot to |exportsObject|.
+    1. Return a new {{Instance}} object whose internal \[[Instance]] slot is set to |instance| and the \[[Exports]] slot to |exportsObject|.
+</div>
+
+<div algorithm>
+  To <dfn>instantate the core of a WebAssembly module</dfn> from a module |module| and imports |imports|, perform the following steps:
```

Comment:
"instantiate the core of a WebAssembly module" is a new term? It's not part of the spec

Typo:
instantate -> instantiate

Reply:
Typo PR is https://github.com/WebAssembly/esm-integration/pull/23.

What about "instantiate the internals of a WebAssembly module" instead?